src/__tests__/role-helpers.js

```diff
@@ -132,15 +131,6 @@ test('getRoles returns expected roles for various dom nodes', () => {
   })
 })
 
-test('logRoles logs expected roles for various dom nodes', () => {
-  expect.addSnapshotSerializer(jestSerializerAnsi)
```

Comment:
I missed this in the previous review, but you could add this to https://github.com/testing-library/dom-testing-library/blob/master/tests/setup-env.js instead and then it'll be applied to all snapshots which is probably a good thing.

Reply:
Will do!  Test is fixed, was overthinking things.. been a long day. 

Adding the serializer to setup-env incoming